app/components/articleSearch/components/searchItem/index.tsx

```diff
@@ -65,136 +51,49 @@ const mockComment: ICommentRecord = recordifyComment({
 });
 const mockComments = List([mockComment, mockComment, mockComment]);
 
-class SearchItem extends React.PureComponent<ISearchItemProps, ISearchItemStates> {
-  private restParagraphElement: HTMLDivElement;
-  private restParagraphElementMaxHeight: number;
-  private restParagraphElementClientHeight: number;
-
-  public constructor(props: ISearchItemProps) {
-    super(props);
-
-    this.state = {
-      isContentOpen: false,
-      isCommentsOpen: false,
-      commentInput: "",
-    };
-  }
-
+class SearchItem extends React.PureComponent<ISearchItemProps, {}> {
+  // private restParagraphElement: HTMLDivElement;
```

Comment:
Remove comment lines.

Reply:
I will use this line after connecting with Social Auth
So I remained this.